lib/membership/member.js

```diff
@@ -242,4 +217,26 @@ Member.isStatusPingable = function isStatusPingable(status) {
     }
 };
 
+/**
```

Comment:
I don't think we use this style comment in this project. I would prefer to keep it consistent.